### DISCUSSION TITLE
fail to ScenegraphLayer + getScale + updateTriggers (bug?)

### DISCUSSION THREAD

**Discussion by eviltik** [visgl/deck.gl#8444] [Q&A] [answered]
https://codepen.io/franck34/pen/wvOwVvO

UpdateTriggers on getScale seem's not working with a ScenegraphLayer.

Did i miss something ?

Bug ?

Ty

**Comment by Pessimistress** [comment]
https://github.com/visgl/deck.gl/issues/8445#issuecomment-1904552426

**Reply by Pessimistress** [reply, marked as answer]
Example: https://github.com/visgl/deck.gl/blob/master/examples/gallery/src/hexagon-layer.html

**Comment by eviltik** [comment]
Codepen updated, feedbacks are welcome (optimizations & best practices).

https://codepen.io/franck34/pen/wvOwVvO

Btw, interesting smooth effect !! (zoom in/out)

**Comment by eviltik** [comment]
![image](https://github.com/visgl/deck.gl/assets/4477890/ee8a574a-d510-4325-b601-e9301bf541ec)

work in progress

**Reply by eviltik** [reply]
![image](https://github.com/visgl/deck.gl/assets/4477890/6445309a-6ee1-48eb-a358-d486bc58431d)
